README.md

```diff
@@ -19,22 +19,11 @@ This repository contains templates for bootstrapping a Rust
    cargo generate ratatui-org/ratatui-template
    ```
 
-3. Choose either the [Simple](#simple-template) or [Async](./async/README.md) template.
-
-## Simple template
-
-The simple template will create the following project structure:
-
-```text
-src/
-├── app.rs     -> holds the state and application logic
-├── event.rs   -> handles the terminal events (key press, mouse click, resize, etc.)
-├── handler.rs -> handles the key press events and updates the application
-├── lib.rs     -> module definitions
-├── main.rs    -> entry-point
-├── tui.rs     -> initializes/exits the terminal interface
-└── ui.rs      -> renders the widgets / UI
-```
```

Comment:
Not sure if we should remove this, I think it makes it less visible.

Reply:
I'm fine with that because the structure is/will be slightly different depending on the chosen template.

Reply:
I moved it to the individual template's `README`. I can imagine us having a lot of different templates, so having the individual folders hold the documentation for their template is consistent and easier to maintain.